docker-compose.yml

```diff
@@ -1,41 +1,30 @@
 version: '3'
 services:
-  zookeeper:
-    image: wurstmeister/zookeeper
-    ports:
-      - "2181:2181"
-    logging:
-      options:
-        max-size: "10m"
-        max-file: "5"
-
   kafka:
-    image: wurstmeister/kafka
+    image: bitnami/kafka:latest
+    hostname: kafka
     ports:
       - "9092:9092"
+    volumes:
+      - "${DOCKER_SHARED_VOLUME}:/bitnami"
```

Comment:
Kafka creates a directory named 'kafka' in the DOCKER_SHARED_VOLUME. In the event that this is set to the project folder, the 'kafka' directory should be added to the .gitignore.

![image](https://github.com/CDOT-CV/jpo-ode/assets/21204351/bb45bf14-ebc9-40da-b561-f989d80a3a65)